devices.js

```diff
@@ -5363,7 +5363,26 @@ const devices = [
         model: 'Mega23M12',
         vendor: 'Dresden Elektronik',
         description: 'ZigBee Light Link wireless electronic ballast',
-        extend: generic.light_onoff_brightness_colortemp_colorxy,
+        meta: {configureKey: 3, multiEndpoint: true},
+        supports: 'on/off, brightness, color temperature, color xy',
+        fromZigbee: [fz.color_colortemp, fz.on_off, fz.brightness],
+        toZigbee: [tz.light_onoff_brightness, tz.light_color_colortemp,
+            tz.ignore_transition, tz.ignore_rate, tz.light_alert,
+            tz.light_brightness_move, tz.light_colortemp_move],
+        endpoint: (device) => {
```

Comment:
I expect only adding endpoint is enough. 

- Why did you remove `extend: generic.light_onoff_brightness_colortemp_colorxy,` seems good, it covers everything you added in supports, fromZigbee and toZigbee
- `multiEndpoint: true` isn't needed (none of the converters use this)
- `configure` also isn't needed as this device doesnt have buttons on it (?). For reporting the reporting feature shoudl be used: https://www.zigbee2mqtt.io/information/report.html

Reply:
You are right; just removing most of my copypasta and leaving just `endpoint` in still works as before.

I updated the PR.

The reason why I did it this way is mostly my incomplete understanding of how `zigbee2mqtt` (or zigbee for that matter) work, so I am currently just randomly prodding at things to find out how they work. Thanks for the guidance, and thanks for your great project!